app/src/main/java/com/creditsaison/loansystem/fragments/AccountNewFragment.java

```diff
@@ -1,47 +1,195 @@
 package com.creditsaison.loansystem.fragments;
 
 
+import android.Manifest;
+import android.content.pm.PackageManager;
+import android.graphics.BitmapFactory;
+import android.net.Uri;
 import android.os.Bundle;
 
+import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
+import androidx.core.app.ActivityCompat;
+import androidx.core.content.ContextCompat;
 import androidx.fragment.app.Fragment;
 import androidx.lifecycle.ViewModelProviders;
 
+import android.os.Environment;
+import android.util.Log;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
 
+import android.content.Intent;
+import android.graphics.Bitmap;
+import android.provider.MediaStore;
+import android.widget.Button;
+import android.widget.ImageView;
+
 import com.creditsaison.loansystem.R;
 import com.creditsaison.loansystem.databinding.FragmentAccountNewBinding;
 import com.creditsaison.loansystem.viewmodel.AccountNewViewModel;
 
+
+import java.io.File;
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Locale;
+
+import pub.devrel.easypermissions.EasyPermissions;
+
 /**
  * A simple {@link Fragment} subclass.
  */
 public class AccountNewFragment extends Fragment {
 
-    private FragmentAccountNewBinding binding;
-    private AccountNewViewModel viewModel;
+        // Define the pic id
+        private static final int pic_id = 123;
+
+        // Define the button and imageview type variable
+        Button btn_add_photo;
+        ImageView client_image;
+        File photoFile = null;
+
+        private FragmentAccountNewBinding binding;
+        private AccountNewViewModel viewModel;
 
     public AccountNewFragment() {
-        // Required empty public constructor
+            // Required empty public constructor
+        }
+
+        @Override
+        public void onCreate(@Nullable Bundle savedInstanceState) {
+            super.onCreate(savedInstanceState);
+
+            viewModel = ViewModelProviders.of(this).get(AccountNewViewModel.class);
+
+        }
+
+        @Override
+        public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                Bundle savedInstanceState) {
+
+            binding = FragmentAccountNewBinding.inflate(inflater, container, false);
+            binding.setViewModel(viewModel);
+
+            btn_add_photo = (Button) binding.getRoot().findViewById(R.id.btn_add_photo);
+            client_image = (ImageView) binding.getRoot().findViewById(R.id.client_image);
+
+            btn_add_photo.setOnClickListener(new View.OnClickListener() {
+                @Override
+                public void onClick(View v) {
+                    captureImage();
+                }
+            });
+
+//            btn_add_photo.setOnClickListener(new View.OnClickListener() {
```

Comment:
Remove blocks of code that are commented out

Reply:
this is out of the current scope for the branch